frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -111,74 +99,23 @@ export const CreateOCSServiceForm = withHandlePromise<
       history.push(
         `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
           OCSServiceModel,
-        )}/${getName(ocsRequestData)}`,
+        )}/${getName(getOCSRequestData())}`,
       );
     });
   };
 
   const handleStorageClass = (sc: StorageClassResourceKind) => {
     setStorageClass(sc);
-    const provisioner: string = sc?.provisioner; // required if user selects 'No Default Storage Class' option
-
-    if (provisioner === NO_PROVISIONER) {
-      setOSDSize(defaultRequestSize.BAREMETAL); // for baremetal environment, set requested capacity as 1 Byte
-    } else {
-      setOSDSize(defaultRequestSize.NON_BAREMETAL);
-    }
+    setOSDSize(defaultRequestSize.NON_BAREMETAL);
   };
 
+  const filterSC = React.useCallback((sc) => sc?.provisioner !== NO_PROVISIONER, []);
```

Comment:
src/components/ocs-install/attached-devices/install.tsx line 49

using the same function can we unify & create function out of it?

Reply:
Also `onlyNoProvSC` function in components/ocs-install/attached-devices/install-lso-sc.tsx  doing the almost same thing.

Reply:
It doesn't make sense to combine both as one is filtering the no provisioner SC from the list and other will return a function for filtering the SC to be passed as a part of `StorageClassDropdown` component.